Updated env_id as per ak_env

Error Message

AssertionError: 1 != 3

Stacktrace

self = <tests.upgrades.test_capsule.Scenario_capsule_sync testMethod=test_pre_user_scenario_capsule_sync>

    @pre_upgrade
    def test_pre_user_scenario_capsule_sync(self):
        """Pre-upgrade scenario that creates and sync repository with
        rpm in satellite which will be synced in post upgrade scenario.
    
    
        :id: preupgrade-eb8970fa-98cc-4a99-99fb-1c12c4e319c9
    
        :steps:
            1. Before Satellite upgrade, Sync a repo/rpm in satellite
    
        :expectedresults: The repo/rpm should be synced to satellite
    
         """
        ak = entities.ActivationKey(organization=self.org_id).search(
            query={'search': 'name={}'.format(self.activation_key)})[0]
        ak_env = ak.environment.read()
        product = entities.Product(
            name=self.prod_name, organization=self.org_id).create()
        self.create_repo()
        repo = entities.Repository(
            product=product.id, name=self.repo_name,
            url=self.repo_url).create()
        repo.sync()
        content_view = entities.ContentView(
            name=self.cv_name, organization=self.org_id).create()
        content_view.repository = [repo]
        content_view = content_view.update(['repository'])
        content_view.publish()
        promote(content_view.read().version[0], ak_env.id)
>       self.assertEqual(content_view.read().environment[0].id, ak_env.id)
E       AssertionError: 1 != 3

tests/upgrades/test_capsule.py:101: AssertionError

--- a/tests/upgrades/test_capsule.py
+++ b/tests/upgrades/test_capsule.py
@@ -98,7 +98,7 @@ class Scenario_capsule_sync(APITestCase):
         content_view = content_view.update(['repository'])
         content_view.publish()
         promote(content_view.read().version[0], ak_env.id)
-        self.assertEqual(content_view.read().environment[0].id, ak_env.id)
+        self.assertEqual(content_view.read().environment[-1].id, ak_env.id)
 
         global_dict = {self.__class__.__name__: {
             'env_name': ak_env.name}}
@@ -214,7 +214,7 @@ class Scenario_capsule_sync_2(APITestCase):
         content_view = content_view.update(['repository'])
         content_view.publish()
         promote(content_view.read().version[0], ak_env.id)
-        self.assertEqual(content_view.read().environment[0].id, ak_env.id)
+        self.assertEqual(content_view.read().environment[-1].id, ak_env.id)
         wait_untill_capsule_sync(self.cap_host)
         org_name = entities.Organization().search(
             query={'search': 'id={}'.format(self.org_id)})[0].label

